<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SearchProDialog</class>
 <widget class="QDialog" name="SearchProDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>901</width>
    <height>455</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>410</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_6">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>871</width>
     <height>241</height>
    </rect>
   </property>
   <property name="title">
    <string>结果列表</string>
   </property>
   <widget class="QTableView" name="searchTableViewPro">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>841</width>
      <height>201</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>861</width>
     <height>121</height>
    </rect>
   </property>
   <property name="title">
    <string>请输入搜索条件（无需全部输入，可部分输入）</string>
   </property>
   <widget class="QLabel" name="prostack_label_2">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>50</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>库存大于：</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="search_prostack_spinBox">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>50</y>
      <width>130</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>10000</number>
    </property>
   </widget>
   <widget class="QLabel" name="search_probatno_label">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>50</y>
      <width>66</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>批     号：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="search_probatno_lineEdit">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>50</y>
      <width>130</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="search_prospec_lineEdit">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>50</y>
      <width>130</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="proname_label_2">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>20</y>
      <width>66</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string> 货品名称：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="search_proname_lineEdit">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>20</y>
      <width>130</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="search_pronum_lineEdit">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>20</y>
      <width>130</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="pronum_label_2">
    <property name="geometry">
     <rect>
      <x>8</x>
      <y>20</y>
      <width>66</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string> 货品编码：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="search_procategory_lineEdit">
    <property name="geometry">
     <rect>
      <x>292</x>
      <y>20</y>
      <width>130</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="procategory_label_2">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>20</y>
      <width>66</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string> 货品大类：</string>
    </property>
   </widget>
   <widget class="QLabel" name="search_prospec_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>66</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string> 货品规格：</string>
    </property>
   </widget>
   <widget class="QPushButton" name="searchProBtn_2">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>90</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string> 搜索</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SearchProDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SearchProDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
